MRI_geom/SPD_fMRI.py

```diff
@@ -0,0 +1,142 @@
+import networkx as nx
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import time
+from sklearn.svm import SVC
+from sklearn.model_selection import KFold
+from sklearn.metrics import precision_score, accuracy_score
+from sklearn.metrics import f1_score, recall_score, confusion_matrix
+import geomstats.spd_matrices_space as spd_space
+SPACE = spd_space.SPDMatricesSpace(dimension=28)
```

Comment:
Two lines between the imports and the start of the code.

Reply:
What is the dimension 28? Maybe give it a name so that it is self explanatory, using a CONSTANT variable?
` NAME_EXPLAINING_WHAT_IT_IS = 28` ?